projects/lzo/project.yaml

```diff
@@ -1,4 +1,5 @@
 homepage: "http://www.oberhumer.com"
+main_repo: 'http://www.oberhumer.com/opensource/lzo'
```

Comment:
also not a repo

Reply:
I've removed this because they do not have a repo: Only a website to download the latest code.